### DISCUSSION TITLE
support DDL and dynamic table adding?

### DISCUSSION THREAD

**Discussion by xiaolianghu** [apache/seatunnel#7226] [General]
Does mysql cdc support DDL and dynamic table adding?

**Comment by davidzollo** [comment]
SeaTunnel mysql cdc connector has supported schema evolution. You can set ` schema-changes.enabled ` is true,  firstly , you can pause the cdc task, then you can `add column` `drop column` `rename column` and `modify column` operation on the task config , at last you can resume the task. you can refer these two docs as follows.
1. `https://seatunnel.apache.org/docs/2.3.9/connector-v2/source/MySQL-CDC` 
2. `https://seatunnel.apache.org/docs/2.3.9/seatunnel-engine/user-command`
